src/main/PriceFeedV2.sol

```diff
@@ -73,6 +73,12 @@ contract PriceFeedV2 is IPriceFeedV2, OwnableUpgradeable {
 		return goodPrice;
 	}
 
+	function getExternalPrice(address _token) external view override returns (uint256) {
+		Oracle memory oracle = oracles[_token];
+		require(oracle.primaryWrapper != address(0), "Oracle not found");
+		return IOracleWrapper(oracle.primaryWrapper).getExternalPrice(_token);

```

Comment:
LGTM.
However, I think it'd be better if we also return `lastUpdate` value from `getExternalPrice()` of `IOracleWrapper`.
So in that case we can update this function logic like this:
1. Get prices from both primary and secondary oracles.
2. If secondary is broken, returns the price from primary oracle.
3. If secondary is alive too, compare prices (using Verificator). If two prices are similar, returns the latest one (by comparing `lastUpdate`)
4. If two prices are not similar, returns price from primary.
5. If primary oracle is broken too, then reverts.

Reply:
If it is not so required, you can ignore.